recipe/build_base.bat

```diff
@@ -47,7 +49,13 @@ if "%DEBUG_C%"=="yes" (
 )
 
 :: AP doesn't support PGO atm?
-set PGO=
+if "%CONDA_FORGE%"=="yes" (
+  set PGO=
+)
+:: PGO is not yet working for Python 3.9:
+:: (CopyPGCFiles target) ->
+::   C:\opt\conda\conda-bld\python-split-3.9.0_7\work\PCbuild\pyproject.props(165,5): error : PGO run did not succeed (no python39!*.pgc files) and there is no data to merge [C:\opt\conda\conda-bld\python-split-3.9.0_7\work\PCbuild\pythoncore.vcxproj]
+:: set PGO=
```

Comment:
Maybe it makes sense to put this in a `conda_build_config.yaml` to make it easier for us both to configure?

Reply:
Locally to the recipe I guess?

Reply:
Yeah that's what I was thinking. Though open to other thoughts

Reply:
I am more than happy to change to this then.

Reply:
Upper case in CBC though or not? Shall I just go with OPTIMIZE and PGO?

Reply:
Cool. No strong preference. Yeah that sounds reasonable. Thanks Ray! 😄

Reply:
Ah the one downside with putting it into CBC is that I cannot differentiate a default between CF and AD in the same file so sharing is more difficult. For example AD uses PGO on Windows but CF cannot at present. I have the same concern about OPTIMIZE where conda-forge have it disabled for ppc64le but I haven't seen the need to do that. Again, this is encoded into the recipe ATM so I can determine the appropriate value at runtime.